### ISSUE TITLE
Linux: Debian (9.0) wire package name: please change! (wire-desktop)

### ISSUE THREAD

**Issue by keypey** [wireapp/wire-desktop#442]
Hi.

Debian 9.0 has "wire" package (see: https://packages.debian.org/stretch/wire).

Please, change wire package name (wire-desktop?).

--
kp.,

**Comment by Strit** [comment]
That does not sound like this Wire. It sounds to be a library of coding protocols.

**Comment by keypey** [comment]
It is true! But Wire ("wire-desktop") [wire_2.11.2685_amd64.deb] has this same name "wire" too!
So: please change [Wire ("wire-desktop") [wire_2.11.2685_amd64.deb]] name to "wire-desktop".

--
kp.,

**Comment by ffflorian** [comment]
I actually second that - it would be just a simple change in the [`package.json`](https://github.com/wireapp/wire-desktop/blob/master/electron/package.json#L2). In the current state it is [quite hard](http://askubuntu.com/questions/326288/install-two-applications-with-the-same-name) to install the "Simple Wire encoding protocol command line tool" on Debian while Wire Desktop is installed (and vice versa).

What do you think @bennyn?

**Comment by lipis** [comment]
What should be the name though.. I think that will be the toughest one.. :)

**Comment by lipis** [comment]
### wire-app

**Comment by lipis** [comment]
### wire-desktop

**Comment by bennyn** [comment]
In terms of consistency `wire-desktop` wins (if you ask me) because it's written all over the place (and it's the GitHub repository name). Looking at the `package.json`, our current value for `name` (which is `wire`) would indicate that our project is a light, fast and flexible JavaScript IoC container (https://www.npmjs.com/package/wire).

**Comment by ConorIA** [comment]
@lipis even with #463, the package is still building for me as `wire_2.11.2721_amd64.deb`. Not sure why, but it seems at some point in the build process the name in electron/package.json goes back to `wire`. Is this just happening on my machine?

**Comment by lipis** [comment]
right.. but is that an actual problem?

**Comment by ConorIA** [comment]
Well, it is insofar as the changes you made do not produce a Debian package called "wire-desktop" unless the name is also changed in `info.json`.

**Comment by ffflorian** [comment]
@lipis 

The problem is not the filename - the package `wire` already [exists in a Debian repository](https://packages.debian.org/stretch/wire):
```
 $ dpkg --info wire_2.11.2685_amd64.deb | grep Package
 Package: wire
 $ dpkg --info wire_0-20161027-0git287d8ca-1_amd64.deb | grep Package
 Package: wire
```

This is not an actual problem until we want to offer automatic updates via a Debian repository (see [Debian Policy 3.1](http://www.debian.org/doc/debian-policy/ch-binary.html#s3.1)) or someone wants to install both packages. Anyways I think it's bad practice. :slightly_smiling_face:

**Comment by lipis** [comment]
I'm with you guys to change it.. I just forgot that we are updating the package name :) Will test tomorrow on how it affects other platforms and if everything is fine, we can update it. My previous update was a bit useless.. the issue remains :D

**Comment by ffflorian** [comment]
Fixed in f01b4332428cbc74787d1a18fe7625d27321a786.
